.github/workflows/deploy-docs-site.yml

```diff
@@ -29,22 +36,47 @@ jobs:
           cache: "yarn"
           cache-dependency-path: docusaurus
 
+
       - name: Run Docusaurus
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./tools/bin/deploy_docusaurus
+        run: |-
+          export SKIP_DEPLOY="no"
+          if [ "${{github.event_name}}" = "pull_request" ]; then
+              export SKIP_DEPLOY="yes"
+          fi
+
+          ./tools/bin/deploy_docusaurus
+
+
+      - name: Notify Slack -- deploy failed
+        if: always() && ( github.event_name == 'push' || github.event_name == 'workflow_dispatch' ) && job.status != 'success'
+        uses: abinoda/slack-action@master
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
+        with:
+          # 'C02TYDSUM8F' => "#dev-deploys'
+          # 'C03BEADRPNY' => presumably '#docs'
```

Comment:
`C032Y32T065` is #docs

![Screenshot 2022-12-16 at 7 40 39 AM](https://user-images.githubusercontent.com/2591516/208110799-b45e237f-94cd-4fba-a08a-80e94b85a775.png)

Reply:
_i cannot check this because this channel is absent in GL slack workspace_

Reply:
_corrected in_ [a4300bb](https://github.com/airbytehq/airbyte/pull/20563/commits/a4300bbbe0af81e3d12f91039c49ace3a2d33bb7)